src/app/webbrowser/BookmarksFoldersViewWide.qml

```diff
@@ -134,34 +134,34 @@ FocusScope {
 
         onActiveFocusChanged: {
             if (activeFocus) {
-                internal.lastFocusedColumn = bookmarksList
+                internal.lastFocusedColumn = bookmarksList;
             }
         }
 
         model: {
             if (!folders.currentItem || !folders.currentItem.model) {
-                return null
+                return null;
             }
 
-            if (folders.currentItem.isAllBookmarksFolder && (bookmarksFoldersViewItem.homeBookmarkUrl.toString() !== "")) {
```

Comment:
here the wrong id bookmarksFoldersViewItem (instead of bookmarksFoldersViewWideItem) was used, that's why it did not work on the tablet even if a homepage was set.